bftengine/src/bftengine/ReplicaImp.cpp

```diff
@@ -1364,7 +1365,7 @@ namespace bftEngine
 					CheckpointMsg* x = new CheckpointMsg(msgSenderId, msgSeqNum, msgDigest, msgIsStable);
 					tableOfStableCheckpoints[msgSenderId] = x;
 
-					LOG_INFO_F(GL, "Node %d added stable Checkpoint message to tableOfStableCheckpoints (message from node %d for seqNumber %" PRId64 ")", (int)myReplicaId, (int)msgSenderId, msgSeqNum);
+					LOG_DEBUG_F(GL, "Node %d added stable Checkpoint message to tableOfStableCheckpoints (message from node %d for seqNumber %" PRId64 ")", (int)myReplicaId, (int)msgSenderId, msgSeqNum);
```

Comment:
I think we should leave this as INFO level. It's a very important message, and infrequent.

Reply:
Thank you for highlighting the importance of this specific message; I did not realize it myself. I will revert this to `INFO`.